OneCommunityGlobal/HighestGoodNetworkApp
# Yubo-Fix Edit link modal style

# Description
### Priority LOW:
<img width="786" alt="Screenshot 2023-08-20 at 1 48 20 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/508c19f0-875f-4063-80f0-568a4645cf54">

## Main changes explained:
- Removed `<input>`, use`<label>` instead.
- import CSS properly.


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. go to another user's profile page→ click links Edit under their pictures
5. verify the styles looks good (remember to add some link to be able to see **delete** button).

## Screenshots or videos of changes:
### Before --  After
<img width="1039" alt="Screenshot 2023-08-20 at 2 01 20 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/983ddfc5-ae46-4ac4-8859-b0507e4cadf8">
